browser-test/src/api_docs.test.ts

```diff
@@ -112,6 +112,112 @@ test.describe('Viewing API docs', () => {
     })
   })
 
+  test('Views active API docs with multiple upload', async ({
```

Comment:
Can we add a TODO and an issue to remove these duplicate tests, and reconcile them with the non-multiple-upload versions, once the feature is launched and the flag is removed?